### PR TITLE
Add increaseAssetSupply transaction

### PR DESCRIPTION
**Pull request by cubismic**
I add increaseAssetSupply transaction

### CODE REVIEW

File: src/models/logic/assettransferinput.ts [hunk 1]
```diff
@@ -50,7 +50,6 @@ export async function createAssetTransferInput(
             }
         });
     } catch (err) {
-        console.log(err);
```
Comment (joojis) [inline src/models/logic/assettransferinput.ts]: Why did you remove this line?

Comment (cubismic) [reply src/models/logic/assettransferinput.ts]: I made a mistake.

File: src/models/logic/utxo.ts [hunk 2]
```diff
@@ -40,7 +40,6 @@ export async function createUTXO(
             blockNumber
         });
     } catch (err) {
-        console.error(err, utxo.assetType.value);
```
Comment (joojis) [inline src/models/logic/utxo.ts]: Same here
